homeassistant/components/vizio/strings.json

```diff
@@ -22,14 +22,14 @@
       },
       "pairing_complete_import": {
         "title": "Pairing Complete",
-        "description": "Your VIZIO SmartCast TV is now connected to Home Assistant.\n\nYour Access Token is '**{access_token}**'."
+        "description": "Your VIZIO SmartCast TV is now connected to Home Assistant.\n\nYour [%key:common::config_flow::data::access_token%] is '**{access_token}**'."
       }
     },
     "error": {
       "host_exists": "VIZIO device with specified host already configured.",
       "name_exists": "VIZIO device with specified name already configured.",
-      "complete_pairing failed": "Unable to complete pairing. Ensure the PIN you provided is correct and the TV is still powered and connected to the network before resubmitting.",
-      "cant_connect": "Could not connect to the device. [Review the docs](https://www.home-assistant.io/integrations/vizio/) and re-verify that:\n- The device is powered on\n- The device is connected to the network\n- The values you filled in are accurate\nbefore attempting to resubmit."
+      "complete_pairing_failed": "Unable to complete pairing. Ensure the PIN you provided is correct and the TV is still powered and connected to the network before resubmitting.",
+      "cant_connect": "[%key:common::config_flow::error::cannot_connect%]"
```

Comment:
Consider changing `"cant_connect"` to `"cannot_connect"` in the future, as that's standard.